supertokens/utils.go

```diff
@@ -181,6 +181,28 @@ func SendNon200Response(res http.ResponseWriter, message string, statusCode int)
 	return nil
 }
 
+func SendNon200ResponseWithPayload(res http.ResponseWriter, payload map[string]interface{}, statusCode int) error {
```

Comment:
This is inconsistent with the node PR. Please see the changes to this file once again in node pr

Reply:
Done